backend/src/test/groovy/com/capstone/moneytree/controller/UserControllerTest.groovy

```diff
@@ -598,12 +592,11 @@ class UserControllerTest extends Specification {
         def response = userController.unfollowUser(user1.getId(), user2.getId())
 
         then: "user1 should have unfollowed user2"
-        //        assert user1.getFollowers().isEmpty()
+        // assert user1.getFollowers().isEmpty()
```

Comment:
Is this not working?

Reply:
This line was already commented out. I just removed the extra space.